airbyte-integrations/connectors/source-tempo/source_tempo/client.py

```diff
@@ -38,7 +38,7 @@ def lists(self, name, url, params, func, **kwargs):
             response = requests.get(f"{self.base_api_url}{url}?limit={params['limit']}&offset={params['offset']}", headers=self.headers)
             data = func(response.json())
             yield from data
-            if len(data) < self.DEFAULT_ITEMS_PER_PAGE:
+            if not self.ENTITIES_MAP[name]["paginated"] or len(data) < self.DEFAULT_ITEMS_PER_PAGE:
```

Comment:
I'm not sure this is the safe way to handle this. As another API repo in python is listening on the `next` field existing in the API response in order to continue. See https://github.com/stanislavulrych/tempo-api-python-client/blob/62418b72b10c625c7fcc9b9d1ea61f09a3c6a09d/tempoapiclient/client.py#L54

I took an example API call for worklogs and can confirm the response object `next` is in there https://tempo-io.github.io/tempo-api-docs/v2.html#worklogs - This is the safest way and easier based on the data return vs. being explicit each time. Also there are chances when data may be more than the default and not have a page and this could cause errors.

Reply:
@bstrawson thanks for explination. All is good 👍

Reply:
@bst code changes are all good, we are just miasing one change - you need to edit the md files too when making changes. Let me know if you need me to point you the connector md with an example

Reply:
@koconder Apologies for missing that. If you could point me in the direction of the md with an example, I'd be grateful.

Reply:
@bstrawson you need to edit this file:
* https://github.com/airbytehq/airbyte/blob/master/docs/integrations/sources/tempo.md (see the bottom)

Reply:
Ok thanks. Done.

Reply:
@bstrawson will get the team to check the integration test results which failed and report back. Should be able to get this review finalized soon